ocsp_status.go

```diff
@@ -76,21 +83,36 @@ func parseOCSPResp(ocspResp []byte, issuerCert *x509.Certificate) {
 	// ocsp.ParseResponse validates signature with issuerCert
 	parsedResp, err := ocsp.ParseResponse(ocspResp, issuerCert)
 	if err != nil {
+		// look at ocsp response errors
 		fmt.Println(string(ocspResp))
 		panic(err.Error())
 	}
-	linter.CheckOCSPResp(parsedResp)
+	linter.LintOCSPResp(parsedResp)
 }
 
 func sendOCSPReq(rootCert *x509.Certificate, issuerCert *x509.Certificate, reqType string, dir string) {
 	httpReq := createOCSPReq(rootCert, issuerCert, reqType)
 
+	
+	startTime := time.Now()
+
 	httpClient := &http.Client{}
 	httpResp, err := httpClient.Do(httpReq)
 	if err != nil {
 		panic(err.Error())
 	}
 
+	endTime := time.Now()
+	limit, err := time.ParseDuration(RespTimeLimit)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	// Verification (source from Apple Lint 08)
+	if (endTime.Sub(startTime) > limit) {
+		fmt.Println("Server took longer than 10 seconds to respond")
```

Comment:
So one thing I tried was setting the constant `RespTimeLimit` to be a `time.Duration` variable, but Go disallowed that since constants cannot be initialized with a function return value, (I tried to do `const RespTimeLimit := time.ParseDuration('10s')` but that didn't work).

Did I understand your question correctly?

Reply:
Well I meant more of something like since limit comes from RespTimeLimit, which looks like a string '10s' can we use RespTimeLimit to do string concatenation here?  Something like `"Server took longer than "+RespTimeLimit+" to respond"`